kube-monitoring/charts/Chart.yaml

```
@@ -8,7 +8,7 @@ maintainers:
 name: kube-monitoring
 sources:
   - https://github.com/cloudoperators/greenhouse-extensions
-version: 0.8.2
+version: 0.8.3
```

That's not necessary, is it? There is no actual change in the chart and it is not reflected in the plugin below. 
```suggestion
Version: 0.8.2
```

---

True, was due to the first intended change. Pushing again to keep `version` with lower `v`